estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
We try to put a return character at the end of the last line in every file.
This way, when a new line is added, the formerly last line is not considered as a part of the diff.